src/agent/command_store/include/command.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <limits>
+#include <string>
+
+namespace command_store
+{
+    enum class Status
+    {
+        SUCCESS,
+        ERROR,
+        INPROCESS,
+        TIMEOUT
+    };
+
+    class Command
+    {
+    public:
+        Command()
```

Comment:
Is this constructor needed?

Reply:
I added it to avoid a warning (uninitialized use of an object). Maybe there's another workaround.